Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -132,5 +133,15 @@ public DBResult<Communication> Update(Communication communication, bool commit =
             this.logger.LogDebug($"Finished updating Communication in DB");
             return result;
         }
+
+        /// <inheritdoc />
+        public List<Communication> GetEmailCommunicationsInNewProcessingOrError()
+        {
+            this.logger.LogTrace($"Getting Communications by Type and Status Code from DB...");
+            List<Communication> retVal = this.dbContext.Communication.Where(c => c.CommunicationTypeCode == CommunicationType.Email && (c.CommunicationStatusCode == CommunicationStatus.New || c.CommunicationStatusCode == CommunicationStatus.Processing || c.CommunicationStatusCode == CommunicationStatus.Error))
```

Comment:
Instead, we can search for not processed.

Reply:
I think in general explicitly list all the statuses that need to be returned/processed is better as what if we introduce a new status in the future that we don't need to return here? Also what about "Pending" communication even we don't have any plan to use this "Pending" status.